staging/vhost-device-gpu/src/device.rs

```diff
@@ -38,7 +38,6 @@ use vhost_user_backend::{VhostUserBackendMut, VringRwLock, VringT};
 use virtio_bindings::{
     bindings::virtio_config::{VIRTIO_F_NOTIFY_ON_EMPTY, VIRTIO_F_RING_RESET, VIRTIO_F_VERSION_1},
     bindings::virtio_ring::{VIRTIO_RING_F_EVENT_IDX, VIRTIO_RING_F_INDIRECT_DESC},
-    virtio_gpu::{VIRTIO_GPU_F_CONTEXT_INIT, VIRTIO_GPU_F_RESOURCE_BLOB},
```

Comment:
I think we should do it the other way around. Use the definitions in the virtio_bindings, and only define things ourselves that aren't in virtio_bindings crate.

Reply:
right, but there are some names in the virtio-binding that are too long to use e.g `virtio_gpu_ctrl_type_VIRTIO_GPU_CMD_GET_DISPLAY_INFO` I would rather use `VIRTIO_GPU_CMD_GET_DISPLAY_INFO`